vendor/k8s.io/kubectl/pkg/cmd/get/get.go

```diff
@@ -263,7 +263,10 @@ func (o *GetOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []stri
 		if outputObjects != nil {
 			printer = &skipPrinter{delegate: printer, output: outputObjects}
 		}
-		if o.ServerPrint {
+
+		outputFormatSpecified := o.PrintFlags.OutputFormat != nil && len(*o.PrintFlags.OutputFormat) > 0
```

Comment:
Close but not quite right, we have several output formats:
```
  -o, --output='': Output format. One of:
json|yaml|wide|name|custom-columns=...|custom-columns-file=...|go-template=...|go-template-file=...|jsonpath=...|jsonpath-file=...
```
only some should not use server printing, but `name`, `wide` will certainly do. I'd check what has changed in get and printers between 1.18 and 1.19 that might have caused this regression.

Reply:
ah right, ok :eyes: